packages/back-end/src/util/secrets.ts

```diff
@@ -22,11 +22,27 @@ export const UPLOAD_METHOD = (() => {
   return "local";
 })();
 
-export let MONGODB_URI =
-  process.env.MONGODB_URI ??
-  (prod ? "" : "mongodb://root:password@localhost:27017/test?authSource=admin");
+let MONGODB_URI = process.env.MONGODB_URI;
+
+if (!MONGODB_URI) {
+  // Check for alternate mongo db environment variables
+  if (
+    process.env.MONGODB_USERNAME &&
+    process.env.MONGODB_PASSWORD &&
+    process.env.MONGODB_HOSTNAME
+  ) {
+    MONGODB_URI = `mongodb://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASSWORD}@${process.env.MONGODB_HOSTNAME}/growthbook`;
```

Comment:
You might as well also add MONGODB_DBNAME and have it default to `growthbook` if it is not set.

Reply:
Good point. I added it.